tests/framework/ROM/pickleTests/tests

```diff
@@ -22,4 +22,10 @@
     input = 'not_a_rom_error.xml'
     expect_err = 'RuntimeError: Pickled object in'
   [../]
+  [./test_external_pickled]
+    type = 'RavenPython'
+    input = '../../../../scripts/externalROMloader.py load_ROM_externally.xml'
```

Comment:
Did this file make it into the PR?

Reply:
Yes. it is in the scripts and in the test folder